internal/resourcepolicies/resource_policies.go

```diff
@@ -0,0 +1,116 @@
+package resourcepolicies
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Action defined as one action for a specific way of backup
+type Action struct {
+	// Type defined specific type of action, currently only support 'skip'
+	Type string `yaml:"type"`
+	// Parameters defined map of parameters when executing a specific action
+	// +optional
+	// +nullable
+	Parameters map[string]interface{} `yaml:"parameters,omitempty"`
```

Comment:
Are the `+optional` and `+nullable` marker useful here?

Reply:
Removed